packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx

```diff
@@ -0,0 +1,131 @@
+
+const SfScrollable = polymorphicForwardRef<typeof defaultScrollableTag, SfScrollableProps>(
+  (
+    {
+      as,
+      direction = SfScrollableDirection.horizontal,
+      buttonsPlacement = SfScrollableButtonsPlacement.blocked,
+      activeIndex,
+      reduceMotion,
+      drag,
+      onDragChange,
+      onScroll,
+      onPrev,
+      onNext,
+      className,
+      wrapperClassNames,
+      previousDisabled,
+      nextDisabled,
+      style,
+      children,
+      slotPreviousButton,
+      slotNextButton,
+      ...attributes
+    },
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    ref,
+  ) => {
+    const Tag = as || defaultScrollableTag;
+
+    const sliderOptions = useMemo(
+      () => ({
+        direction,
+        activeIndex,
+        reduceMotion,
+        drag,
+        onDragChange,
+        onScroll,
+        onPrev,
+        onNext,
+      }),
+      [direction, activeIndex, reduceMotion, drag, onDragChange, onScroll, onPrev, onNext],
+    );
+
+    const { getContainerProps, state, getNextButtonProps, getPrevButtonProps } = useScrollable(sliderOptions);
+
+    const changeDisabledClass = (isDisabled: boolean) =>
+      isDisabled ? '!ring-gray-200 !text-disabled-500' : '!ring-neutral-500 !text-neutral-500';
+    const previousButton = (...buttonClassName: Parameters<typeof classNames>) =>
+      (slotPreviousButton && <div {...getPrevButtonProps()}>{slotPreviousButton}</div>) || (
+        <SfButton
+          variant="secondary"
+          size="lg"
+          className={classNames(
+            buttonClassName,
+            changeDisabledClass(
+              typeof previousDisabled === 'boolean' ? previousDisabled : getPrevButtonProps().disabled,
+            ),
+          )}
+          square
+          slotPrefix={<SfIconChevronLeft />}
+          {...getPrevButtonProps()}
+          disabled={previousDisabled}
+        />
+      );
+
+    const nextButton = (...buttonClassName: Parameters<typeof classNames>) =>
+      (slotNextButton && <div {...getNextButtonProps()}>{slotNextButton}</div>) || (
+        <SfButton
+          variant="secondary"
+          size="lg"
+          square
+          className={classNames(
+            buttonClassName,
+            changeDisabledClass(typeof nextDisabled === 'boolean' ? nextDisabled : getNextButtonProps().disabled),
+          )}
+          slotPrefix={<SfIconChevronRight />}
+          {...getNextButtonProps()}
+          disabled={nextDisabled}
+        />
+      );
+
+    const isHorizontal = direction === SfScrollableDirection.horizontal;
+
+    return (
+      <div
+        className={classNames(
+          'items-center relative',
+          isHorizontal ? 'flex' : 'flex-col h-full inline-flex',
+          wrapperClassNames,
+        )}
+      >
+        {buttonsPlacement === SfScrollableButtonsPlacement.blocked &&
+          previousButton('!rounded-full bg-white', isHorizontal ? 'mr-4' : 'mb-4 rotate-90')}
```

Comment:
Should consider implementing dedicated `z-index` value for buttons as these are rendered on the same level as list elements, 
For example there is a conflict of `absolute` positioned elements (`next/image` position)

<img width="224" alt="image" src="https://user-images.githubusercontent.com/32792205/231111971-55798417-d225-4cf5-92bb-a6a9690a0b3d.png">

Maybe should consider buttons isolation from other list elements.